lib/kitchen/directions/bake_toc.rb

```diff
@@ -26,7 +26,7 @@ def self.v1(book:)
         )
       end
 
-      def self.li_for_unit(unit)
+      def self.li_for_unit(unit, cases: self.cases)
```

Comment:
what does `self.cases` mean here? this seems to be implying that `cases` is a variable or method that exists on this class, but I don't see how that could be

Reply:
@kswanson33 I just pick the solution from https://docs.rubocop.org/rubocop/cops_lint.html#lintcircularargumentreference becuase of Rubocop error (Circular argument reference - `cases`.Lint/CircularArgumentReference(RuboCop)
) when I have just wrote ` def self.li_for_unit(unit, cases: cases)`  I didn't like it either because it meant for me just the same thing you wrote above and i didn't know why it was working exactly either. But I didn't have any other ideas. But I tried to use options here today also (despite we have only one optional parameter here) and it worked. Let me know what you think about this solution or if it will be better to disable Rubocop Lint/CircularArgumentReference? :thinking: